saas/static/js/djaodjin-saas-angular.js

```diff
@@ -642,3 +667,102 @@ metricControllers.controller('metricCtrl',
             $scope.balances = data;
     });
 }]);
+
+revenueControllers.controller('revenueCtrl',
+    ['$scope', '$http', '$filter', 'urls',
+    function($scope, $http, $filter, urls) {
+
+    $scope.endOfMonth = function(date) {
+        return new Date(
+            date.getFullYear(),
+            date.getMonth() + 1,
+            0
+        )
+    };
+
+    $scope.ends_at = $scope.endOfMonth(new Date());
+
+    $scope.tabs = [
```

Comment:
This seems hardcoded copy/paste from the information returned by the Django view. Expected?